roles/zooplus/tasks/globalprotect.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Check if globalprotect client is installed
```

Comment:
Not needed apt have a buildin routine as a preflight check

Reply:
we need a proper .deb package for globalProtect... i leave this on your hands :P

I discovered when you connect via the command line to VPN using globalprotect client, if it connects successfully... it returns you a 1 return code WTF!!!